### ISSUE TITLE
Color picker in editor doesn't allow entering code values

### ISSUE THREAD

**Issue by Rodeo-McCabe** [godotengine/godot#69065]
### Godot version

4.0 beta 5

### System information

Linux

### Issue description

The color picker in the editor has a button next to the hex value LineEdit that allows to switch between entering hex values and entering code values. When switching to code values, the placeholder text is highlighted, but the LineEdit is inactive and you cannot enter text.

### Steps to reproduce

1. Click on some property in the inspector that will bring up the color picker, for example `modulate` on any CanvasItem.
2. Click on the `#` button next to the hex value to switch to code value.
3. Attempt to enter some text.

### Minimal reproduction project

Not necessary.

**Comment by KoBeWi** [comment]
The code label is only for copy-pasting the color.

**Comment by Rodeo-McCabe** [comment]
It doesn't allow to paste data either, it is disabled.

**Comment by KoBeWi** [comment]
I mean you copy the color and paste into e.g. script.

Although maybe this limitation is not necessary. But if it was editable, it would need to parse an expression or something.

**Comment by cengiz-pz** [comment]
It is very hard to edit the hex code in the color picker.  Most of the time, it won't allow editing, but sometimes after clicking back and forth multiple times it is possible to edit the hex value.

**Comment by KoBeWi** [comment]
@cengiz-pz That's new. Do you have any screenshots/videos of that bug? What version are you using?
It sounds like a separate issue from this one, which isn't about hex codes.

**Comment by cengiz-pz** [comment]
> @cengiz-pz That's new. Do you have any screenshots/videos of that bug? What version are you using? It sounds like a separate issue from this one, which isn't about hex codes.

@KoBeWi, I am using Godot Editor 4.2.1.stable on Windows 11.  Is this issue not reproducible in your environment?

**Comment by KoBeWi** [comment]
Yes, the hex field is automatically focused and works every time:

https://github.com/godotengine/godot/assets/2223172/a2895381-6999-48c7-990b-2cb68ef302d0

**Comment by boundlessfaust** [comment]
I am experiencing this issue in the 4.2.1.stable build of the editor. I have discovered that the tooltip seems to be interfering with entering a value in the HEX field but it eventually works if you click on it to hide the tooltip.
![Godot Editor ColorPicker HEX Input](https://github.com/godotengine/godot/assets/84153321/b98ecd05-8fe2-49ac-a4df-1674a0f62fe7)

**Comment by KoBeWi** [comment]
That's #69065

**Comment by Calinou** [comment]
@KoBeWi I think you meant https://github.com/godotengine/godot/issues/79500 :slightly_smiling_face:
